platform/darwin/test/MGLLightTest.mm

```diff
@@ -25,7 +25,10 @@ - (void)testProperties {
     {
         mbgl::style::Light light;
         MGLLight *mglLight = [[MGLLight alloc] initWithMBGLLight:&light];
-        XCTAssertEqual(mglLight.anchor, MGLLightAnchorViewport);
+        
+        NSAssert([mglLight.anchor isKindOfClass:[MGLConstantStyleValue class]], @"mglLight.anchor isn’t a MGLConstantStyleValue.");
```

Comment:
Since this is a test, would it make more sense to use an XCT assertion instead of NSAssert?